app/models/stop.rb

```diff
@@ -284,6 +291,52 @@ def clean_attributes
   end
 end
 
+class StopStation < Stop
+  current_tracked_by_changeset({
+    kind_of_model_tracked: :onestop_entity,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.